lib/closex/mock_client/mock_client.ex

```diff
@@ -367,20 +370,26 @@ defmodule Closex.MockClient do
 
   ## Examples
 
-    > Closex.MockClient.update_opportunity(Closex.MockClient.search_term())
+    > Closex.MockClient.find_leads("foo")
     ...contents of test/fixtures/find_leads.json...
```

Comment:
Perhaps rename this to `find_one_lead.json`?

Reply:
👍